MyPy Error on 5.10.0

Hello,

With the `5.10.0` release we run into a syntax error in `aiogoogle/resource.py:38` when running mypy on our project.

```
[~/projects/foo]$ poetry update aiogoogle
Updating dependencies
Resolving dependencies... (1.7s)

Package operations: 0 installs, 1 update, 0 removals

  • Updating aiogoogle (5.9.0 -> 5.10.0)

Writing lock file
[~/projects/foo]$ poetry run mypy --config-file ./pyproject.toml src/
/Users/username/Library/Caches/pypoetry/virtualenvs/foo-vYsNM9k8-py3.12/lib/python3.12/site-packages/aiogoogle/resource.py:38: error: invalid syntax  [syntax]
Found 1 error in 1 file (errors prevented further checking)
```

Our mypy config is simple

```toml
# pyproject.toml
[tool.mypy]
plugins = ["mypy_django_plugin.main"]
ignore_missing_imports = true
```

We downgraded to `5.9.0` and the issue was fixed

```
[~/projects/foo]$ poetry update aiogoogle
Updating dependencies
Resolving dependencies... (1.7s)

Package operations: 0 installs, 1 update, 0 removals

  • Updating aiogoogle (5.10.0 -> 5.9.0)


[~/projects/foo]$ poetry run mypy --config-file ./pyproject.toml src/
Success: no issues found in 137 source files
```

EDIT: additional context we are using mypy `1.10.1`

--- a/aiogoogle/resource.py
+++ b/aiogoogle/resource.py
@@ -35,7 +35,7 @@ STACK_QUERY_PARAMETER_DEFAULT_VALUE = {"type": "string", "location": "query"}
 MEDIA_SIZE_BIT_SHIFTS = {"KB": 10, "MB": 20, "GB": 30, "TB": 40}
 
 # TODO: etagRequired: {
-#    type: "boolean",  # noqa: F821 (weird error)
+#    should be type: "boolean",  # noqa: F821 (weird error)
 #    description: "Whether this method requires an ETag to be specified. The ETag is sent as an HTTP If-Match or If-None-Match header."
 #    }
 # NOTE: etagRequired is only mentioned once in all of the discovery documents available from Google. (In discovery_service-v1. So, it isn't actually being used)

